test/test-widgets.js

```diff
@@ -230,7 +230,11 @@ test('multipleCheckbox', function (t) {
 test('multipleCheckbox multiple selected', function (t) {
     var w = forms.widgets.multipleCheckbox(),
         field = {
-            choices: { one: 'Item one', three: 'Item three', two: 'Item two' },
+            choices: {
+                one: 'Item one',
+                two: 'Item two',
+                three: 'Item three'
+            },
             value: ['two', 'three']
```

Comment:
nope. just changed it locally. if i change the choices list back they way it was and switch to value keys (that’s what you meant right?) the inputs and labels still appear in order of the choices object (one, three, two).